Src/CSharpier/CodeFormatter.cs

```diff
@@ -4,51 +4,50 @@
 
 namespace CSharpier;
 
-public class CodeFormatter
+public static class CodeFormatter
 {
-    public static LanguageVersion LanguageVersion = LanguageVersion.Preview;
+    internal static readonly LanguageVersion LanguageVersion = LanguageVersion.Preview;
 
-    public static string Format(string code, CodeFormatterOptions? options = null)
+#pragma warning disable RS0026 // I know these are breaking and I can't seem to get Unshipped/Shipped happy with me doing it
```

Comment:
Why don't you just add a new set of overloads without the `options` arg? That way it doesn't have to be an optional parameter and might resolve this nicely.

Reply:
It is complaining about the options but that isn't what changed. I changed the return type and it seems to be confused by that. Adding those overloads may get it happy, but then it may just complain again if I remove the overloads in the next release.

This seemed like the easiest way past this and I just need to remove the warning after 0.23.0 comes out.

Reply:
I'll take your word for it :shrug: